src/main/fox_hud.c

```diff
@@ -161,14 +163,14 @@ Gfx D_800D1A40[] = {
 
 void func_80084930(f32 arg0, f32 arg1, s32 arg2) {
     u8* D_800D1A58[] = {
-        D_3000000,
-        D_3000000,
-        D_3000000,
+        D_arwing_3000000,
+        D_arwing_3000000,
+        D_arwing_3000000,
```

Comment:
these aren't from the same file, they are just the same segment address. Same below

Reply:
The first one is from ast_arwing, the other ones are from ast_landmaster and ast_blue_marine